src/libs/SearchUIUtils.ts

```diff
@@ -446,7 +445,11 @@ function getTaskSections(data: OnyxTypes.SearchResults['data']): TaskListItemTyp
     return Object.keys(data)
         .filter(isReportEntry)
         .map((key) => {
-            const taskItem = data[key] as SearchTask;
+            const taskItem = data[key] as SearchListItem;
+            if (!isTaskListItemType(taskItem)) {
+                return taskItem;
+            }
```

Comment:
```suggestion
            const taskItem = data[key] as SearchTask;
```

Let's revert this too, my PR will merge with these changes